servers/physics_2d/godot_joints_2d.cpp

```diff
@@ -170,12 +173,40 @@ bool GodotPinJoint2D::pre_solve(real_t p_step) {
 	if (B && dynamic_B) {
 		B->apply_impulse(P, rB);
 	}
+	real_t i_sum_local = A->get_inv_inertia();
+	if (B) {
+		i_sum_local += B->get_inv_inertia();
+	}
+	i_sum = 1.0 / (i_sum_local);
+	if (enable_limits && B) {
+		Vector2 diff_vector = B->get_transform().get_origin() - A->get_transform().get_origin();
+		diff_vector = diff_vector.rotated(-initial_angle);
+		real_t dist = diff_vector.angle();
+		real_t pdist = 0.0;
+		if (dist > angular_limit_upper) {
+			pdist = dist - angular_limit_upper;
+		} else if (dist < angular_limit_lower) {
+			pdist = dist - angular_limit_lower;
+		}
+		real_t error_bias = powf(1.0 - 0.15, 60.0);
```

Comment:
```suggestion
		real_t error_bias = Math::pow(1.0 - 0.15, 60.0);
```
Use the built-in library for math